### DISCUSSION TITLE
Gotek model sfrkc30.at2  with AT32F415 can't get it to work with Atari ST

### DISCUSSION THREAD

**Discussion by vmaxuser** [keirf/flashfloppy#880] [Q&A]
Hi to all!
I own  3 Goteks.
1 model SFRC9220 with STM32F105 which works fine.
2 models SFRKC30.AT with a AT32F415 which have all the  same problem...
All models flashed with latest firmware. 
Oled + rotary encoder seems to work normal but I can't load programs from usb stick.
btw JC is present on the PCB

![SFRKC30 AT2](https://github.com/keirf/flashfloppy/assets/18097088/0b9b4a8f-cbad-4fde-8b55-3b8b01bdc292)

When I switch on my Atari, I see the tracks on the display increment with 5 steps at a time  10.0 , 15.0 , 20.0 ,  etc. but that's all that I see.
Anyone with a solution?

thanks a lot for your help or ideas

**Comment by keirf** [comment]
Why do you have MO and JC jumpered?

**Comment by vmaxuser** [comment]
Just to try every combination.
I tried with and without jumpers
The incremental 5 steps are only with a jumper at MO.

I started with the same jumper combination as the old model which works fine. 

 

 

Van: Keir Fraser ***@***.***> 
Verzonden: woensdag 13 maart 2024 16:42
Aan: keirf/flashfloppy ***@***.***>
CC: vmaxuser ***@***.***>; Author ***@***.***>
Onderwerp: Re: [keirf/flashfloppy] Gotek model sfrkc30.at2 with AT32F415 can't get it to work with Atari ST (Discussion #880)

 

Why do you have MO and JC jumpered?

**Comment by keirf** [comment]
Jumper at S0 only is certainly correct, matching your old Gotek. Are you loading ".ST" image files?

**Comment by keirf** [comment]
I just tested my own SFRKC30.AT2 on an Amiga with S0 only and default configuration, and that works fine with firmware 3.42.

Please make sure you are using firmware 3.42 (not any version 4.x) and also no FF.CFG on your USB drive, and reset any cached configuration by pressing both buttons for a few seconds with no USB stick inserted until the display changes.

**Reply by vmaxuser** [reply]
I have firmware version 3.42 installed and tried all possible jumper combinations. 
Jumper setting on the photo is one of them.
I use only hfe files
I use the same usb stick in the working and non working Gotek.

**Reply by keirf** [reply]
Try an st file.

**Reply by vmaxuser** [reply]
With ST file exact the same problem

**Reply by keirf** [reply]
No further advice I'm afraid, sorry.

**Reply by vmaxuser** [reply]
Thanks for your support.

**Comment by AlexLevytskyi** [comment]
unfortunately I have the same problem for Atari ST and SFRKC30.AT3. I tried several firmwares (3.34, 3.35, 3.42...).  415 is not for Atari ST((

**Reply by keirf** [reply]
But original STM32F105 does work? And have you tried an AT32F435 Gotek?

**Reply by AlexLevytskyi** [reply]
435 is ok! 
Problem only with 415.

**Reply by keirf** [reply]
I feel it is probably related to the RAM size. Are you using regular .ST images?

**Reply by AlexLevytskyi** [reply]
Yes. I use ST images.

I think it's possible. Device try to start image, indicator changes numbers, but it halt before finish line)) So you may be right.

**Reply by AlexLevytskyi** [reply]
> I feel it is probably related to the RAM size. Are you using regular .ST images?

I received second gotek with at415. And know what?... It's ok with flashfloppy))) it's working without problem!

The first one (doesn't work) has a label 'sfrm72-fu-dl'. I think it's new blacklist candidate)
